web-ui/src/main/resources/catalog/components/admin/uiconfig/partials/uiconfig.html

```diff
@@ -436,6 +436,78 @@ <h3>{{type}}</h3>
           {{('ui-' + key + '-help') | translate}} </p>
       </div>
 
+      <div data-ng-switch-when="switcherProjectionList">
+        <label class="control-label">{{('ui-' + key) | translate}}</label>
+        <p class="help-block"
+           data-ng-show="(('ui-' + key + '-help') | translate) != ('ui-' + key + '-help')">
+          {{('ui-' + key + '-help') | translate}} </p>
+        <table class="table table-striped">
```

Comment:
You need the resolutions because you have to define them on some projections, or at least on some of the cases I tried, it failed if we didn't define them. And world extent is required, at least on the current gn code for loading layers.

I will work on improving the settings UI.